Miki.Discord.Common/Utils/Mention.cs

```diff
@@ -11,6 +11,8 @@ public enum MentionType
         CHANNEL,
         EMOJI,
         ANIMATED_EMOJI,
+        MENTION_ALL,
```

Comment:
Maybe USER_ALL(_ONLINE) would work better with the others

Reply:
Makes sense 😄